packages/nodejs/src/instrumentation/redis/index.ts

```diff
@@ -82,7 +82,7 @@ export const instrument = (
   mod: RedisModule,
   tracer: Tracer
 ): Plugin<RedisModule> => ({
-  version: ">= 3.0.0",
+  version: ">=3.0.0 <4.0.0",
```

Comment:
I'm looking at the other operators we can use to do this version restriction. Would the tilde operator `~3` also work? Or `^3.x`?

https://github.com/npm/node-semver#tilde-ranges-123-12-1

Reply:
Yes, `~3` should work.

I use this for testing: https://semver.npmjs.com/